github/ci/prow-deploy/files/jobs/kubevirt/hostpath-provisioner/hostpath-provisioner-presubmits.yaml

```
@@ -33,3 +33,65 @@ presubmits:
           resources:
             requests:
               memory: "29Gi"
+  - name: pull-hostpath-provisioner-sanity
+    skip_branches:
+      - release-\d+\.\d+
+    annotations:
+      fork-per-release: "true"
+    always_run: false
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 1h
+      grace_period: 5m
+    max_concurrency: 6
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror: "true"
+    spec:
```

We need to define the cluster this job should run on.
```suggestion
    cluster: prow-workloads
    spec:
```

---

Okay, I don't care much which cluster this runs on, if possible I would avoid the same cluster that the kubevirtci jobs run on since I don't need baremetal. If prow-workloads is good for that, I will add that.

---

Ok, then `cluster: phx-prow` would be better in this case.